### ISSUE TITLE
assert!(a == b) to assert_eq!(...) intention looses message

### ISSUE THREAD

**Issue by TimNN** [intellij-rust/intellij-rust#2073]
If the "Convert to assert_eq!" intention is applied to code like `assert!(1 == 1, "msg");` the result is `assert_eq!(1, 1);`, with the message being lost.

```
Plugin Version: 0.2.0.2214-nightly

IntelliJ IDEA 2017.3 (Ultimate Edition)
Build #IU-173.3727.78, built on November 21, 2017
JRE: 1.8.0_152-release-1024-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.1
```

**Comment by Undin** [comment]
Looks like that it was fixed in #2044

**Comment by TimNN** [comment]
Apparently I'm selectively blind. I totally didn't see the existing issue when searching for the problem >.<.
